npc/jobs/2-2/crusader.txt

```diff
@@ -223,6 +209,27 @@ prt_castle,45,169,5	script	Senior Crusader	4_M_CRU_OLD,{
 	else if(CRUS_Q >= 1 && CRUS_Q <= 3) {
 		mes "Have you proven your determination with the task I have given you, or do you possess the items proving that you have received your calling?";
 		next;
+		if (countitem(Patriotism_Marks) >= 1 && countitem(Sacred_Marks) >= 1) {
+			mes "Ah...";
+			mes "I see that you have been called to become a Crusader. We are assured of your will, but now we must test your capabilities.";
+			next;
+			mes "[Michael Halig]";
+			mes "Meet with Moorenak Miyol who is training in the underground dungeon of the Prontera Castle. Go, and speak with him first.";
+			next;
+			delitem Patriotism_Marks, 1;
+			delitem Sacred_Marks, 1;
+			CRUS_Q = 4;
+			if (questprogress(3006)) {
+				changequest 3006,3009;
+			} else if (questprogress(3007)) {
+				changequest 3007,3009;
+			} else {
+				changequest 3008,3009;
```

Comment:
I'm just following the behavior the script is already using at line 259 (Master).
I could change that if actually needed, although I didn't quite get your reasoning for now.
While it seems to be possible to shorten down lines by using erasequest(3005, 3010) or completequest(3005,3010) (I hope I read the documentations correctly, it says *between*, not if it's excluding or including.).

Aegis script uses changequest at this part btw. (Implying our changequest is an exact imitation of theirs)